src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/BeneficialOwnersStatementType.java

```diff
@@ -11,13 +13,13 @@ public enum BeneficialOwnersStatementType {
    BeneficialOwnersStatementType(String beneficialOwnersStatement) {
         this.beneficialOwnersStatement = beneficialOwnersStatement;
     }
-
-    public static BeneficialOwnersStatementType findByBeneficialOwnersStatementTypeString(String beneficialOwnersStatement) {
-        for (BeneficialOwnersStatementType type: values()) {
-            if(type.beneficialOwnersStatement.equals(beneficialOwnersStatement)) {
-                return type;
-            }
-        }
-        return null;
-    }
+   @JsonCreator
```

Comment:
Blank line missing above the annotation and I think that @markpit also suggested a comment here as to what this achieves.